crates/sage/src/database.rs

```diff
@@ -598,6 +618,8 @@ mod test {
             decoy_tag: "rev_".into(),
             generate_decoys: false,
             fasta: "none".into(),
+            prefilter: false,
```

Comment:
I guess we should add a few more test cases covering the different search space and chunking types to ensure that the results are identical.

Reply:
Fully agreed! Any volunteers to do this;)...?